drakrun/drakrun/main.py

```diff
@@ -164,15 +165,28 @@ def _get_dll_run_command(pe_data):
 
         return None
 
-    def _get_start_command(self, extension, sample):
+
+    @staticmethod
+    def _get_office_file_run_command(extension, file_path):
+        start_command = 'start '
+        if d_office.is_office_word_file(extension):
+            start_command += 'winword.exe /t %f'
+        else:
+            start_command += 'excel.exe /t %f'
+
+        outer_macros = d_office.get_outer_nodes_from_vba_file(file_path)
+        for outer_macro in outer_macros:
+            start_command += f' /m{outer_macro}'
```

Comment:
Is it safe against shell injection? I think it would be good to have whole this method working on lists and doing `subprocess.list2cmdline` at the end.